### ISSUE TITLE
New feature: A link with "_blank" in target attribute

### ISSUE THREAD

**Issue by keysona** [trentm/python-markdown2#213]
For example,
```
[Ref](http://www.example.com)
```
to
```
'<p><a target="_blank"  href="http://www.example.com">Ref</a></p>'
```

This can open a  linked document in a new tag.

I want to support this, can I ?

I think it useful for me.

**Comment by AlJohri** [comment]
+1 for supporting `target="_blank"`

**Comment by keysona** [comment]
#212

**Comment by nicholasserra** [comment]
Fixed in #212
